h5bench_patterns/h5bench_write.c

```diff
@@ -1147,10 +1224,19 @@ main(int argc, char *argv[])
             read_time_val(params.compute_time, TIME_US) * (params.cnt_time_step - 1);
         printf("Total emulated compute time: %.3lf s\n", total_sleep_time_us / (1000.0 * 1000.0));
 
+        // Report total write size
         double total_size_bytes = NUM_RANKS * local_data_size;
         value                   = format_human_readable(total_size_bytes);
         printf("Total write size: %.3lf %cB\n", value.value, value.unit);
 
+        float compression_ratio = total_size_bytes / COMPRESS_INFO.total_compressed_size;
```

Comment:
What is the initial value of `total_compressed_size`? Could this cause an exception by dividing by zero. Could we put this calculation inside the if?

Reply:
It was initialized to 0. Yes we should add a check.